stream_alert/shared/backoff_handlers.py

```diff
@@ -17,41 +17,64 @@
 from stream_alert.shared import LOGGER
 
 
-def backoff_handler(details):
+def backoff_handler(debug_only=True):
```

Comment:
can we rename to just `debug`? I was a little confused what the purpose of `only` is in that var name since it just does one or the other

Reply:
I chose `debug_only` here because `debug` seems misleading to me... for instance, if I call a function:

`run_this(debug=True)`

That gives me the impression that *_extra logging_* will occur (like a verbose mode), when in fact thats not at all what happens.

So `debug_only` seemed more clear to me in that `run_this(debug_only=True)` implies it would only have an affect if debug mode is enabled.

Does that make sense?

Reply:
makes sense